frontend/src/containers/FormToSubmit.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { addSurvey } from '../actions/action-creators';
+
+
+
+
+let FormToSubmit = ({ dispatch }) => {
+    let input
+
+    return (
+        <div className='col-md-12 form-to-submit'>
+            <p className='form-to-submit__header lead'>Count of models to show:</p>
+        <form onSubmit={e => {
+          e.preventDefault()
+          if (!input.value.trim()) {
+            return
+          }
+          dispatch(addSurvey(input.value))
+          input.value = ''
+        }}>
+          <input onChange={e =>{
```

Comment:
Don't inline complex event handlers into JSX markup. Define component methods and use them as event handlers.